### ISSUE TITLE
Bun install is broken for @mui/lab

### ISSUE THREAD

**Issue by galkin** [oven-sh/bun#4701]
### What version of Bun is running?

1.0.0+822a00c4d508b54f650933a73ca5f4a3af9a7983

### What platform is your computer?

Darwin 22.6.0 arm64 arm

### What steps can reproduce the bug?

`bun install @mui/lab`

### What is the expected behavior?

bun should be able to install @mui/lab. it has more than 1m downloads per week.

### What do you see instead?

```
❯ bun install @mui/lab
bun add v1.0.0 (822a00c4)


error: package "@mui/x-tree-view" with tag "https://pkg.csb.dev/mui/mui-x/commit/1f23b33d/@mui/x-tree-view" not found, but package exists
```

### Additional information

@mui/lab has package.json with such dependency:
```
    "@mui/x-tree-view": "https://pkg.csb.dev/mui/mui-x/commit/1f23b33d/@mui/x-tree-view"
```
It is a valid tar file, but without an extension. I guess, because of that bun can not install it.

**Comment by ncovercash** [comment]
Looks like this is a duplicate of #4666.

**Comment by galkin** [comment]
closed as duplicate of #4666
